packages/eslint-plugin-ckeditor5-rules/lib/rules/use-require-for-debug-mode-imports.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const DEFAULT_ALIAS_IMPORT = /import { default as (.*) } from (.*)?;/;
+const NAMED_ALIAS_IMPORT = /import { (.*) as (.*) } from (.*)?;/;
+const NAMED_IMPORT = /import { (.*) } from (.*)?;/;
+const NAMESPACE_ALIAS_IMPORT = /import .* as (.*) from (.*)?;/;
+const DEFAULT_IMPORT = /import (.*) from (.*)?;/;
+const SIDE_EFFECT_IMPORT = /import (.*)?;/;
+
+const DESTRUCTURED_ALIAS_REPLACE = 'const { $1: $2 } = require( $3 );';
+const DESTRUCTURED_REPLACE = 'const { $1 } = require( $2 );';
+const MODULE_REPLACE = 'const $1 = require( $2 );';
+const DEFAULT_REPLACE = 'const $1 = require( $2 ).default;';
+const SIDE_EFFECT_REPLACE = 'require( $1 );';
+
+module.exports = {
+	meta: {
+		type: 'problem',
+		docs: {
+			description: 'Disallow using the `import` keyword for modules used in debug mode.',
+			category: 'CKEditor5'
+		},
+		fixable: 'code',
+		messages: {
+			'usingImportNotAllowed': 'Using import with `@if CK_DEBUG_*` keyword is not allowed. Use `require()` instead.'
+		},
+		schema: []
+	},
+	create( context ) {
+		return {
+			Program() {
+				const source = context.getSourceCode();
+				const comments = source.getAllComments();
+
+				for ( const comment of comments ) {
+					if ( debugCommentDoesNotContainImport( comment.value ) ) {
+						continue;
+					}
+
+					context.report( {
+						node: comment,
+						messageId: 'usingImportNotAllowed',
+						fix( fixer ) {
+							const newCommentValue = getNewComment( comment.value );
+							const newComment = addCommentCharacters( comment.type, newCommentValue );
+
+							return fixer.replaceText( comment, newComment );
+						}
+					} );
+				}
+			}
+		};
+	}
+};
+
+/**
+ * @param {String} [str='']
+ * @returns {Boolean}
+ */
+function debugCommentDoesNotContainImport( str = '' ) {
+	return !/@if CK_DEBUG.* \/\/ import/.test( str );
```

Comment:
Edge case: multiple debug imports could be aligned using white space 👇🏻 

```
// @if CK_DEBUG //     import ...
// @if CK_DEBUG_FOO // import ...
// @if CK_DEBUG_X //   import ...
```

```suggestion
	return !/@if CK_DEBUG.* \/\/\s*import/.test( str );
```